.github/workflows/reviewdog.yml

```
@@ -38,6 +38,40 @@ jobs:
           flake8 --docstring-convention=all | \
             reviewdog -f=pep8 -name=flake8 \
               -tee -reporter=github-check -filter-mode nofilter
+  mypy:
+    name: mypy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python 3
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
```

```suggestion
          python-version: 3.9
```


This fails on merged version of code because `functools.cache` was new in 3.9